### PR TITLE
Add more info about use cases and tutorials to contributor guide

### PR DESCRIPTION
**Pull request by marklewin**
## Description

Adding more info to help people decide which is the best format for their "how to". Also describe the process for creating a Use Case.

## Deploy Notes

**Note**: The translations for the new menu items in the nav bar are configured in ADP, so [PR 3299](https://github.com/Nexmo/nexmo-developer/pull/3299) needs to be merged _before_ this one

### CODE REVIEW

**Comment by marklewin** [comment]
Would love @tbedford to have a quick look at this! There are still changes that need to be made in Nexmo Developer for this to work properly, but basically I've done the following:

- Added a new "Tutorials vs Use Cases" guide to help contributors decide which one they need
- Renamed the Structure > Guides section to Structure > Content Types and added a Use Cases section in there
- Moved the actual implementation instructions for tutorials and use cases to its own section (like we have for code snippets)

**Comment by tbedford** [comment]
If I go to https://station-pr-354.herokuapp.com/contribute I get an error. 

> There are still changes that need to be made in Nexmo Developer for this to work properly

What would need to be done to Nexmo Developer? Is there something in addition to just updating it to use a new version of Station?

**Comment by marklewin** [comment]
Yes, I have to navigate to
https://station-pr-354.herokuapp.com/contribute/overview directly for this
to work :/

**Comment by tbedford** [comment]
It looks good. I note the missing translation in the sidebar, and [images](https://station-pr-354.herokuapp.com/contribute/tutorials-and-use-cases/use-cases). It looks like these need to be on the Nexmo Developer side? I think we are going to have to look at pulling the Contribution Guide out altogether to make it easier to maintain. It could have its own repo/pipeline possibly.

**Review by tbedford — APPROVED** [review]

**Comment by marklewin** [comment]
Thanks @tbedford ... I believe the images should be rendered from within station? That's where [the existing images](https://station-pr-354.herokuapp.com/contribute/structure/content-types/tutorials) seem to be coming from? But for whatever reason, the [new ones I have added](https://station-pr-354.herokuapp.com/contribute/tutorials-and-use-cases/use-cases) aren't displaying. @bencgreenberg ... any idea why this might be happening?

The menu translations will have to happen on the NDP side though.

> It looks good. I note the missing translation in the sidebar, and [images](https://station-pr-354.herokuapp.com/contribute/tutorials-and-use-cases/use-cases). It looks like these need to be on the Nexmo Developer side? I think we are going to have to look at pulling the Contribution Guide out altogether to make it easier to maintain. It could have its own repo/pipeline possibly.

**Comment by bencgreenberg** [comment]
Yes, the menu translations happen on the NDP/ADP side of things.

**Comment by bencgreenberg** [comment]
@marklewin Where are those images physically stored?

**Comment by marklewin** [comment]
> Yes, the menu translations happen on the NDP/ADP side of things.

Thanks @bencgreenberg But I am I correct in saying that the images should be rendered from Station?

**Comment by marklewin** [comment]
> @marklewin Where are those images physically stored?

`/station/lib/nexmo_developer/public/assets/images/contributing`

**Comment by tbedford** [comment]
@marklewin I don't see the new image files in the PR

File: lib/nexmo_developer/app/views/contribute/tutorials-and-use-cases/use-cases.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+> **Wait!** Are you sure that a Use Case is the right format for what you want to achieve? Please read our [Tutorials and Use Cases guide](/contribute/guides/tutorials-and-use-cases) before continuing.
+
+Create the file for your Use Case in the appropriate language folder for your locale in the `_use_cases` directory, e.g. `_use_cases/en`, for English.
+
+Begin your Use Case with a Level 1 heading (`#`) and suitable introductory paragraph.
+
+Each main topic within your Use Case should be a Level 2 heading (`##`). You can create subheadings under each of the main topics, but please try to not go beyond Level 3 (`###`) unless absolutely necessary. Please avoid anything beyond a Level 4 (`####`) heading.
+
+![Creating a Use Case](/assets/images/contributing/create-a-uc.png)
```
Comment (bencgreenberg) [inline lib/nexmo_developer/app/views/contribute/tutorials-and-use-cases/use-cases.md:20]: This image is not in the directory

Comment (marklewin) [reply lib/nexmo_developer/app/views/contribute/tutorials-and-use-cases/use-cases.md:20]: @bencgreenberg Thanks - the images directory is in `.gitignore` ... is there a reason for that?

Comment (bencgreenberg) [reply lib/nexmo_developer/app/views/contribute/tutorials-and-use-cases/use-cases.md:20]: I believe the intention is to strip Station of specific portal content and move most of that to instances of Station (VBC, ADP, etc.)

**Pull request merged**